app/package.json

```diff
@@ -31,6 +32,7 @@
   },
   "scripts": {
     "start": "PORT=3001 react-scripts start",
+    "start-windows": "set PORT=3001 && react-scripts start",
```

Comment:
What is the purpose of line 35?

Reply:
I have same question.

Reply:
That is for starting the project on windows environment. Setting up the Port value on Windows machine.